### ISSUE TITLE
loss decreasing with validation dice 0.0

### ISSUE THREAD

**Issue by DISAPPEARED13** [MIC-DKFZ/nnUNet#1035]
Hi there,
I'm working 5-classes SSL with nnUNet based, and modified the original GenericUNet as student model, 
but I found that after a little bit strange for **validation average foreground dice** in `run_online_evaluation` all got 0.0 as result, though my train loss and valiation loss still got decreasing.
```
2022-04-22 08:42:09.298326: 
epoch:  0 
2022-04-22 08:55:02.819094: train loss : 1.1987 
2022-04-22 08:58:55.729233: validation loss : 0.7238 
2022-04-22 08:58:55.729984: Average global foreground Dice: [0.0, 0.0, 0.0, 0.0754, 0.0] 
2022-04-22 08:58:55.730217: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-04-22 08:58:56.614424: lr: 0.000998 
2022-04-22 08:58:56.614645: This epoch took 1007.315773 s
 
2022-04-22 08:58:56.614700: 
epoch:  1 
2022-04-22 09:04:44.566778: train loss : 0.9323 
2022-04-22 09:07:56.618958: validation loss : 0.6121 
2022-04-22 09:07:56.619883: Average global foreground Dice: [0.0, 0.0, 0.0, 0.0, 0.0] 
2022-04-22 09:07:56.619995: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-04-22 09:07:57.683688: lr: 0.000995 
2022-04-22 09:07:57.683963: This epoch took 541.069213 s
 
2022-04-22 09:07:57.684021: 
epoch:  2 
2022-04-22 09:13:00.137630: train loss : 0.8466 
2022-04-22 09:21:53.298978: validation loss : 0.5612 
2022-04-22 09:21:53.299664: Average global foreground Dice: [0.0, 0.0, 0.0, 0.0, 0.0] 
2022-04-22 09:21:53.299770: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-04-22 09:21:54.293819: lr: 0.000993 
2022-04-22 09:21:54.294035: This epoch took 836.609952 s
 
2022-04-22 09:21:54.294113: 
epoch:  3 
2022-04-22 09:27:28.795778: train loss : 0.8172 
2022-04-22 09:30:08.974333: validation loss : 0.4837 
2022-04-22 09:30:08.975077: Average global foreground Dice: [0.0, 0.0, 0.0, 0.0, 0.0] 
2022-04-22 09:30:08.975175: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-04-22 09:30:10.172712: lr: 0.000991 
2022-04-22 09:30:10.173045: This epoch took 495.878867 s
 
2022-04-22 09:30:10.173128: 
epoch:  4 
2022-04-22 09:34:58.706732: train loss : 0.7928 
```
 
And I tried to print prediction label, and every pixel set to 0. I notice that the estimate might not be exact but is it normal?
Or... Is it because that my learning rate or some hyper-parameters value set too high?

Thanks a lot!
Best,

**Comment by DISAPPEARED13** [comment]
if anybody know that is it normal that dice goes to zero at the start of training? Thanks alot!
